src/test/java/characterpicker/infrastructure/CharacterParserTest.java

```diff
@@ -8,7 +8,7 @@
     @Test
     void shouldThrowExceptionWhenSpacesSeparatedByCommaIsProvided() {
         //GIVEN
-        String emptyInput = "   ,   ";
+        String emptyInput = "   ,   ; ";
```

Comment:
Chyba wygodniej by to się używało kiedy jakbyśmy chcieli pominąć podawanie completion marków to w ogóle nie podajemy `;`, a podajemy go tylko wtedy kiedy mają być jakieś completion marki

Reply:
Jeżeli chcesz przetestować co się stanie jak do completion marków dasz spacje na końcu to powinno to być sprawdzone w oddzielnym teśćie.